src/tools/compiletest/src/runtest.rs

```diff
@@ -2029,11 +2029,19 @@ impl<'test> TestCx<'test> {
             rustc.args(&["-A", "unused"]);
         }
 
+        let mut lld = false;
```

Comment:
```suggestion
        let mut fuse_ld_lld= false;
```
You are specifically detecting a gcc-like linker using `-fuse-ld=lld` here, and also rely on it when passing `-Wl,--threads=1` which also needs a gcc-like linker.
LLD in general can be used in other ways, MSVC targets, for example, use `lld` directly as `-C linker`.